packages/components/src/panel/types.ts

```diff
@@ -36,3 +41,73 @@ export type PanelRowProps = {
 	 */
 	children: React.ReactNode;
 };
+
+export type PanelBodyProps = {
+	/**
+	 * Props that are passed to the Button component in the PanelBodyTitle within the panel body.
+	 *
+	 * @default {}
+	 */
+	buttonProps?: ButtonProps;
+	/**
+	 * The content to display within the panel body.
+	 *  If the children is a Function, it will be called with an object with the opened property and return its value.
+	 */
+	children?: React.ReactNode | ( ( {} ) => React.ReactNode );
+	/**
+	 * The CSS class to apply to the wrapper element.
+	 */
+	className?: string;
+	/**
+	 * An icon to be shown next to the PanelBody title.
+	 */
+	icon?: JSX.Element;
+	/**
+	 * Whether or not the panel will start open.
+	 */
+	initialOpen?: boolean;
+	/**
+	 * A function that is called when the user clicks on the PanelBody title after the open state is changed.
+	 *
+	 * @default true
```

Comment:
The default value for the `onToggle` prop i[s assigned when destructuring the prop](https://github.com/WordPress/gutenberg/blob/db1449e741312ad48d8c412767117cd00dafb834/packages/components/src/panel/body.tsx#L35), and in this case is `noop`

```suggestion
	 * @default noop
```

Reply:
Ah, that was a copy/paste error, thanks for catching it! 🙇‍♀️